scene/main/viewport.cpp

```diff
@@ -1266,6 +1266,9 @@ void Viewport::_gui_show_tooltip() {
 
 	Window *window = gui.tooltip_popup->get_parent_visible_window();
 	Rect2i vr = window->get_usable_parent_rect();
+	if (gui.tooltip_popup->is_embedded()) {
+		vr = Rect2i(Point2i(), window->size);
+	}
```

Comment:
There's no point to the first line calculating `vr` if it's overridden afterwards, so this should likely be `if () {} else {}`.

Reply:
True, updated.